admob/src/ios/banner_view_internal_ios.mm

```diff
@@ -79,11 +79,19 @@
   AdRequest *request_copy = new AdRequest;
   *request_copy = request;
   dispatch_async(dispatch_get_main_queue(), ^{
-    // A GADRequest from an admob::AdRequest.
-    GADRequest *ad_request = GADRequestFromCppAdRequest(*request_copy);
+    // Create a GADRequest from an admob::AdRequest.
+    AdMobError error = kAdMobErrorNone;
+    GADRequest *ad_request = GADRequestFromCppAdRequest(*request_copy, &error);
     delete request_copy;
-    // Make the banner view ad request.
-    [banner_view_ loadRequest:ad_request];
+    if(ad_request==nullptr) {
+      if(error==kAdMobErrorNone) {
+        error = kAdMobErrorInternalError;
+      }
+      CompleteLoadFuture(error, "");
```

Comment:
Might be nice to also have an error string passed back from GADRequestFromCppAdRequest too, rather than it just LogError-ing. Same in interstitial.

Reply:
(also a default error message for Internal Error)

Reply:
Done.